code/game/objects/structures/construction_consolse/construction_console.dm

```diff
@@ -0,0 +1,126 @@
+//Aux base construction console
+/mob/camera/ai_eye/remote/base_construction
+	name = "construction holo-drone"
+	move_on_shuttle = 1 //Allows any curious crew to watch the base after it leaves. (This is safe as the base cannot be modified once it leaves)
+	icon = 'icons/obj/mining.dmi'
+	icon_state = "construction_drone"
+	var/obj/machinery/computer/camera_advanced/base_construction/linked_console
+
+/mob/camera/ai_eye/remote/base_construction/Initialize(mapload, var/linked_cam_console)
+	. = ..()
+	linked_console = linked_cam_console
+
+/mob/camera/ai_eye/remote/base_construction/setLoc(t)
+	var/area/curr_area = get_area(t)
+	if(!linked_console.allowed_area || istype(curr_area, linked_console.allowed_area))
+		return ..()
+
+/mob/camera/ai_eye/remote/base_construction/relaymove(mob/living/user, direction)
+	dir = direction //This camera eye is visible as a drone, and needs to keep the dir updated
+	return ..()
+
+/obj/item/construction/rcd/internal //Base console's internal RCD. Roundstart consoles are filled, rebuilt cosoles start empty.
+	name = "internal RCD"
+	max_matter = 600
+	no_ammo_message = "<span class='warning'>Internal matter exhausted. Please add additional materials.</span>"
+	delay_mod = 0.5 //Bigger container and faster speeds due to being specialized and stationary.
+
+/obj/machinery/computer/camera_advanced/base_construction
+	name = "generic base construction console"
+	desc = "An industrial computer integrated with a camera-assisted rapid construction drone."
+	networks = list("ss13")
+	var/obj/item/construction/rcd/internal/RCD //Internal RCD. The computer passes user commands to this in order to avoid massive copypaste.
+	circuit = /obj/item/circuitboard/computer/base_construction
+	off_action = new/datum/action/innate/camera_off/base_construction
+	jump_action = null
+	var/list/datum/action/innate/construction_actions
+	//Number of different special structures that are in stock
+	var/list/structures = list()
+	var/obj/machinery/computer/auxiliary_base/found_aux_console //Tracker for the Aux base console, so the eye can always find it.
+	icon_screen = "mining"
+	icon_keyboard = "rd_key"
+	light_color = LIGHT_COLOR_PINK
+	var/area/allowed_area
```

Comment:
the newly created vars should be below ones already defined

Reply:
Yup, will be fixing this and some related issues tomorrow